param/cobalt.yaml

```diff
@@ -160,3 +160,8 @@ sensor:
             x: 0
             y: 0
             z: 0
+hydrophones:
+    positions:
+        x: 3
+        y: 3
+        z: 3
```

Comment:
These correspond to delta, epsilon, and zeta in the derivation. Aka, they represent how long down the orthogonal axes the hx, hy, and hz hydrophones reside at